vault/logical_system_user_lockout.go

```diff
@@ -157,16 +139,15 @@ func (b *SystemBackend) getLockedUsersResponses(ctx context.Context, mountAccess
 	return totalCounts, lockedUsersResponse, nil
 }
 
-// getMountAccessorsLockedUsers returns the locked users for all the mount_accessors of locked users for a namespace
-// they are sorted in the decreasing order of locked users
-// returns the total locked users for the namespace and  locked users response for every mount_accessor for a namespace that has locked users
-func (b *SystemBackend) getMountAccessorsLockedUsers(ctx context.Context, mountAccessors []string, lockedUsersPath string) (int, []*ResponseMountAccessors, error) {
+// getMountAccessorsLockedUsers returns the locked users for all the mountAccessors
+// of locked users for a namespace. Result is sorted in the desc order of locked users.
+func (b *SystemBackend) getMountAccessorsLockedUsers(ctx context.Context, view BarrierView, mountAccessors ...string) (int, []*ResponseMountAccessors, error) {
```

Comment:
Nit: could be just `logical.Storage`
```suggestion
func (b *SystemBackend) getMountAccessorsLockedUsers(ctx context.Context, view logical.Storage, mountAccessors ...string) (int, []*ResponseMountAccessors, error) {
```